import-service/src/importProductsFile/importProductsFile.ts

```diff
@@ -0,0 +1,42 @@
+import { APIGatewayProxyHandler } from "aws-lambda";
+import { S3 } from "aws-sdk";
+
+import { REGION, BUCKET, HTTP_CODES, CORS_HEADERS } from "../constants";
+
+export const importProductsFile: APIGatewayProxyHandler = async event => {
+  console.log("importProductsFile was called with event: ", event);
+
+  try {
+    const s3 = new S3({ region: REGION });
+
+    const fileName = event.queryStringParameters.name;
```

Comment:
Maybe  it is better to send 400 status code when there is no name

Reply:
Agree) I'll make changes